src/tiles/grid/account-tile.ts

```diff
@@ -2,22 +2,140 @@ import { css, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { TileModel } from '../../models';
 
+import accountIcon from './icons/account';
+import favoriteFilledIcon from './icons/favorite-filled';
+import reviewsIcon from './icons/reviews';
+import uploadIcon from './icons/upload';
+
 @customElement('account-tile')
 export class UserTile extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
   render() {
     return html`
-      <h1>User</h1>
-      <h2>${this.model?.title}</h2>
+      <div class="outter-holder">
+        <div class="inner-holder">
+          <div class="title-holder">
+            <h1>${this.model?.identifier}</h1>
+          </div>
+          <div class="avatar-holder">
+            <div
+              class="avatar"
+              style="background: url('https://archive.org/services/img/${this
+                .model?.identifier}'); background-position: center;"
+            ></div>
+          </div>
+          <div class="credit-holder">
+            <div class="archivist-since">
+              <h3>Archivist Since</h3>
+            </div>
+            <div class="year-holder">
+              <h3>${this.model?.addeddate?.getFullYear()}</h3>
+            </div>
+          </div>
+          <div class="status-holder">
+            <div class="patron">${accountIcon}</div>
```

Comment:
I would make these classes more descriptive, like `patron-icon` (could also become an `id`). `patron` sounds like the user themselves.